extensions/amp-brid-player/0.1/test/validator-amp-brid-player.html

```diff
@@ -36,22 +36,30 @@
       layout="responsive"
       width="480" height="270">
   </amp-brid-player>
-  <!-- Alternate valid example using a playlist rather than video. -->
+  <!-- Alternate valid example using a playlist. -->
   <amp-brid-player
       data-partner="264"
       data-player="4144"
       data-playlist="13663"
       layout="responsive"
       width="480" height="270">
   </amp-brid-player>
+  <!-- Alternate valid example using a outstream. -->
```

Comment:
The actual `validator-amp-brid-player.protoascii` needs to be updated as well. `data-outstream` sould be listed as an attributed and `mandatory_oneof:` values should be updated to include it (the Travis test is failing due to this)